src/app/services/api.service.ts

```diff
@@ -183,28 +170,29 @@ export class ApiService extends CacheModule {
 
 	private async lockOrWait(resource) {
 		if (!this.awaitingResponses[resource]) {
-			console.log('Locking ', resource)
 			this.awaitingResponses[resource] = new Mutex()
 		}
 		await this.awaitingResponses[resource].acquire()
 	}
 
-	private async unlock(resource) {
-		console.log('Unlocking  ', resource)
-
+	private unlock(resource) {
 		this.awaitingResponses[resource].release()
 	}
 
-	async isNotMainnet(): Promise<boolean> {
-		const test = (await this.networkConfig).net != ''
+	isNotEthereumMainnet(): boolean {
+		const test = this.networkConfig.net != ''
 		return test
```

Comment:
**Bug:** This check returns true for Gnosis as well.

Reply:
This is okay, I renamed it to isNotEthereumMainnet

Reply:
Sorry, I meant this returns false for Gnosis as well i.e. Gnosis is handled as Ethereum's mainnet by this function.

Reply:
Has been solved here: https://github.com/gobitfly/eth2-beaconchain-explorer-app/pull/255/files#diff-654ea4aabcdd84b489045ca408d33a859f0f18c051ec3cb15801baca9cf3919aR183